plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -126,23 +124,43 @@ class GeneralServiceFormSection extends Component {
     }
 
     return (
-      <p>
-        {'Need to run a service with multiple containers? '}
-        <a className="clickable" onClick={this.handleOpenConvertToPodModal}>
-          Add another container
-        </a>.
-      </p>
+      <div className="pod pod-narrow pod-short">
```

Comment:
Don't like how `pod pod-narrow pod-short` add padding here. Is it by design?

![screen shot 2017-01-27 at 12 24 34](https://cloud.githubusercontent.com/assets/186223/22369363/b08121a4-e48b-11e6-9213-b7cc67fddc76.png)